packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
```

Comment:
If you are able to could you also update this one to be a range, please? i.e. `>=2.2.1` like the browserslist one above.

Reply:
fixed.